documentation/source/quickstart.rst

```diff
@@ -138,9 +138,31 @@ Installing a package manager really helps things out here.
 
 `Brew <https://brew.sh/>`_ seems to be the most popular, so we'll assume you have that installed.
 
-.. code-block::bash
+.. code-block:: bash
     
     $> brew install python icarus-verilog gtkwave
+
+Installing cocotb
+=================
+
+Cocotb can be installed by running:
+
+.. code-block:: bash
+
+    $> pip install cocotb
+
+Or from repository:
+
+.. code-block:: bash
+
+    $> git clone https://github.com/potentialventures/cocotb
+    $> pip install ./cocotb 
+
+And for development:
+
+.. code-block:: bash
+
+    $> pip install -e ./cocotb 
```

Comment:
- In almost all distributions "pip" defaults to Python 2, while "pip3" uses Python 3. I think we should promote the usage of Python 3 and hence recommend using "pip3" by default.
- Ubuntu, SUSE and other large distributions require the "--user" flag to install Python packages without root permissions. This should at least be mentioned as option, since I've seen many people fail on that step on shared PCs where they don't have root permissions. The downside is, unfortunately, that users need to check if "~/.local/bin" is in their PATH for `cocotb-config` to work, which typically is not the case.

    Something like
    ```
    cocotb-path || echo 'PATH=~/.local/bin:$PATH' >> ~/.bashrc
   ```
   might be a recommendation to add to the docs.

- This is the quickstart guide: it should only give *one recommended* option to install cocotb -- and I'd argue that installing cocotb from a PyPi package is the way to go here. Could you put the other two installation options into another document?

Reply:
I tied my best
- recommend pip3
- direct to pip user guide for details 
- leave how to install development version (not sure where else put this)